src/shared/styles/components/public-homepage.css

```
@@ -0,0 +1,56 @@
+.public-homepage__content {
+  width: 100%;
+  color: var(--text-primary);
+  text-align: center;
+}
+
+.public-homepage h2 {
+  margin: 0;
+  font-size: 2rem;
+  line-height: 1.1;
+
+  color: var(--color-primary);
```

Stylelint (17.7.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)